src/openrct2/actions/SetCheatAction.hpp

```diff
@@ -772,4 +780,17 @@ DEFINE_GAME_ACTION(SetCheatAction, GAME_COMMAND_CHEAT, GameActionResult)
         auto parkSetParameter = ParkSetParameterAction(isOpen ? ParkParameter::Open : ParkParameter::Close);
         GameActions::ExecuteNested(&parkSetParameter);
     }
+
+    void CreateDucks(int count) const
+    {
+        for (int i = 0; i < count; i++)
+        {
+            // 100 attempts at finding some water to create a few ducks at
+            for (int32_t attempts = 0; attempts < 100; attempts++)
```

Comment:
Is there any reason to not use pre-increment instead of post?

Reply:
For primitives there's no difference. The compiler is smart enough to see that the return value is not used and that there is no need to create a copy.

Reply:
This code was copied and nothing changed except the variable name, most of the code consists doing post and not pre increment. I would agree that this might have benefits if it were a hot path but this is not even close.

Reply:
I know and don't think it'll make a difference here (or maybe on most cases), but then there's no real reason to use post if not interested in the old value. Anyway, thanks for answering folks :)

Reply:
Yeah but there is also consistency which is already hard to on this mixed C/C++ environment :)